src/solver/conv/conv_ocl_dir2Dfwd.cpp

```diff
@@ -41,6 +43,18 @@ using ProblemDescription = miopen::conv::ProblemDescription;
 bool ConvOclDirectFwd::IsApplicable(const ExecutionContext& ctx,
                                     const ProblemDescription& problem) const
 {
+// Disable this solver due to random GPU memory access faults on gfx11 and gfx12
+#if WORKAROUND_MIOPEN_1392
```

Comment:
```suggestion
#if WORKAROUND_LWPMIOPEN_1392
```

See the first comment